app/src/main/java/com/poklad/androidifystore/domain/usecases/UseCaseSuspend.kt

```diff
@@ -1,5 +1,9 @@
 package com.poklad.androidifystore.domain.usecases
 
-interface UseCaseSuspend<in Parameter, out Result> {
-    suspend fun execute(params: Parameter): Result
-}
\ No newline at end of file
+import kotlinx.coroutines.flow.Flow
+
+interface UseCaseSuspend<Parameter, Result> {
+//        suspend fun execute(params: Parameter): Flow<Resource<Result>>
+    suspend fun execute(params: Parameter): Flow<Result>
```

Comment:
don't return `flow` please. instead return result

Reply:
please call me and explait this step, or please send me example with this useCase principal